test/AutoRest.TestServer.Tests/LowLevel/ModelsTypeSpecTests.cs

```diff
@@ -1,4 +1,4 @@
-// Copyright (c) Microsoft Corporation. All rights reserved.
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
```

Comment:
Others looks good! Let's revert this change and merge the latest code then we are good to go.

Reply:
fixed